test/server/typegen.ts

```diff
@@ -629,659 +629,6 @@ test('typegen: typescript', async () => {
   )
 })
 
-test('typegen w/ one-to-one relationships', async () => {
```

Comment:
This test was a duplicate with: https://github.com/supabase/postgres-meta/pull/964/files#diff-a9ec34b3928a1f6c1c476fc768d833dbe06c573bc6b9282719574f04f0b4d523L1285